server/util/log/log.go

```diff
@@ -126,11 +136,15 @@ func Configure(level string, enableFileName, enableStructured bool) error {
 			return err
 		}
 	}
+	zerologMu.Lock()
 	zerolog.SetGlobalLevel(intLogLevel)
```

Comment:
This one is already threadsafe: https://github.com/rs/zerolog/blob/v1.21.0/globals.go#L94